DSL.Ruuter/GET/training_status.yml

```diff
@@ -0,0 +1,8 @@
+get_status:
+  call: http.get
+  args:
+    url: http://ml-service:8000/status
```

Comment:
What endpoint is `http://ml-service:8000/status`?

Reply:
This is the endpoint which is hosted on our ML Service responsible for training a model. It returns the current status of training.